unittests/test_autodetect.py

```diff
@@ -119,3 +133,13 @@ def test_remote_autodetect(remote_exec_ctx):
         pytest.skip('job submission not supported')
 
     autodetect.detect_topology()
+
+
+def test_remote_custom_autodetect(remote_custom_exec_ctx):
+    # Test that a dummy topo.json file is created if a command to do
+    # so is specified as custom command for remote detection
```

Comment:
At the moment we are only triggering the autodetection, but cannot tell why the topo.json was created (since the reframe command will be added in the end). This is fine if it's not possible to check the script, but maybe update the comment to reflect that? 🤔

Reply:
With my suggestion of only requiring the reframe installation commands, this unit test would only need to check that the installation command was executed. We could for example have a dummy installation command `touch foo.txt` and here we could check that `foo.txt` exists.